tox.ini

```
@@ -18,6 +18,10 @@ tox_pip_extensions_ext_venv_update = true
 tox_pyenv_fallback = false
 
 [testenv]
+envdir =
+    {tests,coverage}: {toxworkdir}/monotest
```

```suggestion
    {tests,coverage}: {toxworkdir}/tests
```

But I'm not sure about using `envdir` to make one testenv use the same venv as another, anyway. It's unclear to me from the tox docs what `envdir` is for and whether it's really meant to be used in this way. It's possible that it is meant to be. But regardless it does break at least one thing:

When both the testenv's are run in parallel, it now crashes. For example using tox's `-p` for parallelism, if the `monotest` venv hasn't been created yet, will crash:

```
tox -p all -e tests,coverage
```

Or using make's `-j` to do the same thing:

```
make -j 2 test coverage
```

Or simply running them as two separate `tox` or `make` commands in two separate shells at the same time, etc.

I know that it doesn't make sense to ever run tests and coverage at the same time since coverage depends on the result of tests. But that suggests that tests and coverage should actually be one testenv. And still I don't think we should combine their venvs in this way as it might encourage using this method to combine other venvs as well, and because it might also break other tox things besides parralelism.